Source/DATASource.m

```diff
@@ -113,6 +113,11 @@ - (NSInteger)tableView:(UITableView *)tableView
 
 - (NSString *)tableView:(UITableView *)tableView
 titleForHeaderInSection:(NSInteger)section {
+    if ([_delegate respondsToSelector:@selector(dataSource:titleForHeaderInSection:)]) {
```

Comment:
Can you do `self.delegate` instead of `_delegate`? I try to avoid direct global access unless it happens in the `init` method.